zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3711,6 +3882,62 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/UserResponse"
+    RoleCreateRequest:
+      type: "object"
+      properties:
+        name:
+          type: "string"
+    RoleCreateResponse:
+      type: "object"
+      properties:
+        roleKey:
+          description: The key of the created role
+          type: "integer"
+          format: "int64"
+    RoleUpdateRequest:
+      type: object
+      properties:
+        changeset:
+          $ref: "#/components/schemas/RoleChangeset"
+      required:
+        - changeset
+    RoleChangeset:
+      type: object
+      properties:
+        name:
```

Comment:
🔧 Let's provide a `description` for this parameter as well. This should explain what this does, e.g. changing the name of the role. It appears trivial but at first sight I had to think about it for a second, if this changes only the name of the role itself (so, just a "cosmetic" change) or has any further implications.